lib/client-assets.php

```diff
@@ -937,8 +937,9 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	 *
 	 * @param bool|array $allowed_block_types Array of block type slugs, or
 	 *                                        boolean to enable/disable all.
+	 * @param object                          The post resource data.
```

Comment:
This `@param` comment missing the name of the argument.

(It's weird that the linter didn't yell about this.)

Reply:
It's not a function, so it's tricky for both human and computer. I updated :)